openedx/core/djangoapps/user_authn/views/tests/test_login.py

```diff
@@ -1179,29 +1179,18 @@ def test_invalid_credentials(self):
         })
         self.assertHttpBadRequest(response)
 
-        # Invalid email address
```

Comment:
We're all of these test deletions accidental?

Reply:
yes, these were accidental. Thanks for noticing. I reverted them.